client/me/security-ssh-key/security-ssh-key.tsx

```diff
@@ -0,0 +1,132 @@
+/* eslint-disable no-nested-ternary */
+import { CompactCard, LoadingPlaceholder } from '@automattic/components';
+import styled from '@emotion/styled';
+import { createInterpolateElement } from '@wordpress/element';
+import { useI18n } from '@wordpress/react-i18n';
+import { useDispatch } from 'react-redux';
+import DocumentHead from 'calypso/components/data/document-head';
+import FormattedHeader from 'calypso/components/formatted-header';
+import HeaderCake from 'calypso/components/header-cake';
+import Main from 'calypso/components/main';
+import PageViewTracker from 'calypso/lib/analytics/page-view-tracker';
+import { recordTracksEvent } from 'calypso/state/analytics/actions';
+import { errorNotice, successNotice } from 'calypso/state/notices/actions';
+import { AddSSHKeyForm } from './add-ssh-key-form';
+import { ManageSSHKeys } from './manage-ssh-keys';
+import { useAddSSHKeyMutation } from './use-add-ssh-key-mutation';
+import { useDeleteSSHKeyMutation } from './use-delete-ssh-key-mutation';
+import { useSSHKeyQuery } from './use-ssh-key-query';
+
+const SSHKeyLoadingPlaceholder = styled( LoadingPlaceholder )( {
+	':not(:last-child)': {
+		marginBlockEnd: '0.5rem',
+	},
+} );
+
+const Placeholders = () => (
+	<>
+		{ Array( 5 )
+			.fill( null )
+			.map( ( _, i ) => (
+				<SSHKeyLoadingPlaceholder key={ i } />
+			) ) }
+	</>
+);
+
+export const SecuritySSHKey = () => {
+	const { data, isLoading } = useSSHKeyQuery();
+	const dispatch = useDispatch();
+	const { __ } = useI18n();
+
+	const addSSHKey = useAddSSHKeyMutation( {
+		onMutate: () => {
+			dispatch( recordTracksEvent( 'calypso_ssh_key_add_request' ) );
```

Comment:
TODO: Confirm event.

Reply:
Do we need to track requests? We'll be most interested in successful key additions and removals. Tracking failures might come in handy in case of breakages or if something unexpected happens. We could derive requests from those two numbers?